src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -22,6 +23,7 @@
       ]
     },
     {
+      "id" : "9ddd72fa-e32e-4ed7-b475-32d4b19f6b70",
```

Comment:
Will it not cause any problems if we use the same `UUID` for all contacts when testing an address book? Especially if we use the same sample data to test the creation of meetings?